packages/meditrak-app/app/assessment/selectors.js

```diff
@@ -109,3 +118,80 @@ export const getQuestion = (state, questionId) => state.assessment.questions[que
 export const getAnswers = state => state.assessment.answers;
 
 export const getAnswerForQuestion = (state, questionId) => getAnswers(state)[questionId];
+
+const getArithmeticResult = (state, config) => {
+  const { formula, valueTranslation, defaultValues = {}, text = '' } = config.calculated;
+  const values = {};
+  const variables = getVariables(formula);
+  let translatedText = text;
+
+  variables.forEach(questionIdVariable => {
+    let answer = getAnswerForQuestion(state, questionIdVariable) || 0;
+
+    if (
+      valueTranslation &&
+      valueTranslation[questionIdVariable] &&
+      valueTranslation[questionIdVariable][answer] !== undefined
+    ) {
+      answer = valueTranslation[questionIdVariable][answer];
+    }
+
+    if (answer === undefined) {
+      answer = defaultValues[questionIdVariable];
+    }
+
+    values[questionIdVariable] = answer || 0;
```

Comment:
```suggestion
    values[questionIdVariable] = answer !== undefined ? answer : 0;
```
In case we want the answer to be `false`...which seems unlikely in an arithmetic question now I think about it